### PR TITLE
WingpanelInterface: Set durations with constants

### LINKED ISSUE #250: Replace Granite.Services.Settings with GLib.Settings

### PR DESCRIPTION
**Pull request by danrabbit**
Fixes #250

### CODE REVIEW

**Comment by jeremypw** [comment]
I notice that all three gala settings accessed here - `snap-duration`, `minimize-duration` and `workplace-switch-duration` - are all marked as "deprecated and ignored" .  So we need to keep them here?

**Comment by danrabbit** [comment]
You're right! This was done in https://github.com/elementary/gala/pull/513

Updated to use constants.

File: wingpanel-interface/BackgroundManager.vala [hunk 1]
```diff
@@ -105,31 +108,31 @@ public class WingpanelInterface.BackgroundManager : Object {
...
         });
 
         window.notify["minimized"].connect (() => {
-            check_for_state_change (AnimationSettings.get_default ().minimize_duration);
+            check_for_state_change (MINIMIZE_DURATION);
         });
 
         window.workspace_changed.connect (() => {
-            check_for_state_change (AnimationSettings.get_default ().minimize_duration);
+            check_for_state_change (MINIMIZE_DURATION);
```
Comment (jeremypw) [inline wingpanel-interface/BackgroundManager.vala]: This is correctly based on master, but is it right that the `workspace_changed` signal closure uses `MINIMIZE_DURATION" and not`"WORKPLACE_SWITCH_DURATION`?   This can be left for another PR (if required).

Comment (danrabbit) [reply wingpanel-interface/BackgroundManager.vala]: Oh wow. Thank you for actually reading this! I feel dumb for not noticing this :sweat_smile:
